compiler/testData/ir/irText/classes/initValInLambda.kt

```diff
@@ -0,0 +1,9 @@
+// IGNORE_BACKEND: ANY_FIR
```

Comment:
Great, thank you!

Reply:
Yes, that's ok.

Reply:
Would you consider using the same custom ignore directive which we used for the FIR black box tests ([change](https://github.com/JetBrains/kotlin/pull/2800/commits/84d77d126c8c5c2bea0aa62c795a797f223786d4#diff-3e9a3103a63a111fbcef46ce4a8033eeR488)), `// IGNORE_BACKEND_FIR`? That is, you would put `// IGNORE_BACKEND_FIR: ANY` here. For that change, I drew inspiration from the custom ignore directive for multi-module.

One advantage would be it'd be slightly easier to find disabled tests for FIR using the same directive. Another is that you don't need to add a new backend in the enum which is not really a backend. 🙂